### PR TITLE
Lowercasing all filename ref to spdx-spec

### PR DESCRIPTION
**Pull request by bact**
To fix warnings in https://github.com/spdx/spdx-spec/pull/1052

### CODE REVIEW

**Review by goneall — APPROVED** [review]
LGTM - Note - once this is merged, we need to merge the associated spec PR

**Review by kestewart — APPROVED** [review]

**Comment by kestewart** [comment]
Going ahead and merging this, so that https://github.com/spdx/spdx-spec/pull/1052 will validate.

**Pull request merged**
